### DISCUSSION TITLE
Welcome to MCP Discussions!

### DISCUSSION THREAD

**Discussion by StorytellerCZ** [Meteor-Community-Packages/organization#55] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Discuss incorporating new packages into the organization.
  * Coordinate on development and improvements across MCP.

Please do note that for general Meteor discussion you should use the [official Meteor Forums](https://forums.meteor.com).

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by frankwo1** [comment]
Hi @StorytellerCZ , I am a developer and have been using MeteorJS since about 2016. What is the process for adding packages into MCP? In specific, Iron-Router would be a great package to update and maintain as it has been untouched since 2017.

**Reply by StorytellerCZ** [reply]
Right now we are not accepting any as there aren't enough maintainers to manage what we got already.

**Reply by StorytellerCZ** [reply]
Also in regards to Iron-Router, it has been said for years to migrate to Flow Router which has maintained branch.

**Reply by dr-dimitru** [reply]
@frankwo1 [flow-router](https://github.com/veliovgroup/flow-router) ;)

**Comment by frankwo1** [comment]
Thanks, You guys are doing a major necessity for MeteorJS by keeping the ecosystem from getting too stale --- as far as Iron-Router, I prefer it to Flow-Router as I am still working with Blaze for my app --- I don't really see a good reason to change until I am ready to move my app to React

**Reply by mitar** [reply]
I have been using my [fork of flow router](https://github.com/peerlibrary/flow-router) with Blaze for quite some time (not that it matters that it is a fork, what I mean is that flow router works with Blaze).

**Reply by dr-dimitru** [reply]
@frankwo1 [its wiki](https://github.com/veliovgroup/flow-router/wiki) and [quick-start guide](https://github.com/veliovgroup/flow-router/blob/master/docs/quick-start.md) is [all about Blaze](https://github.com/veliovgroup/flow-router/blob/master/docs/templating.md)

**Reply by frankwo1** [reply]
TYSM
Frank
